domainbed/scripts/train.py

```diff
@@ -248,6 +255,12 @@ def save_checkpoint(filename):
                 results[name+'_f1'] = f1
                 results[name+'_nacc'] = non_overlap_class_acc
                 results[name+'_oacc'] = overlap_class_acc
+            
```

Comment:
Make the tsne plots per domain and include the domain in the name of the saved file (e.g. `tsne_plot_{domain}_{step}.png` but you can combine all the data in one csv file like you are doing now

Reply:
if we have a plot per domain, then wouldn't it be not possible to assess domain invariance?